docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.md

```diff
@@ -75,6 +75,18 @@ npm install --save-dev babel-plugin-styled-components @types/node @types/react t
 
 [@types/node](https://www.npmjs.com/package/@types/node) and [@types/react](https://www.npmjs.com/package/@types/react) contain TypeScript type definitions for Node.js and React.js respectively.
 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+> You can view a list of all the tested package versions in the [package.json file on the Step 1 branch](https://github.com/bigcommerce/sample-app-nodejs/blob/step-1-app-foundation/package.json) of this sample app's repo.
```

Comment:
👍 thanks for linking to the package file.

qq: what does `theme: info` look like here?

Reply:
same as the stencil docs ref you linked -- blue. https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil#installing-on-linux